README.md

```diff
@@ -1,45 +1,188 @@
-							 .tox|\.venv|\.svn|_build|buck-out|build|dist)]
-  -h, --help                 Show this message and exit.
-  -V, --version              Show the version and exit.
-  -v, --verbose              Turns on debug-level logging.
+  -l, --line-length INT  Lines longer than INT will be wrapped.  [default: 88]
+  --check                Don't write the files back, just return the status.
+                         Return code 0 means nothing would change. Return code
+                         1 means some files would be reformatted. Return code
+                         123 means there was an internal error.
+
+  -d, --diff             Don't write the files back, just output a diff for
+                         each file to stdout.
+
+  --compact-diff         Same as --diff but only shows lines that would change
+                         plus a few lines of context.
+
+  --include PATTERN      A regular expression that matches files and
+                         directories that should be included on recursive
+                         searches.  An empty value means all files are
+                         included regardless of the name.  Use forward slashes
+                         for directories on all platforms (Windows, too).
+                         Exclusions are calculated first, inclusions later.
+                         [default: (\.smk$|^Snakefile)]
+
+  --exclude PATTERN      A regular expression that matches files and
+                         directories that should be excluded on recursive
+                         searches.  An empty value means no paths are
+                         excluded. Use forward slashes for directories on all
+                         platforms (Windows, too). Exclusions are calculated
+                         first, inclusions later.  [default: (\.snakemake|\.eg
+                         gs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_
+                         build|buck-out|build|dist)]
+
+  -c, --config PATH      Read configuration from PATH. By default, will try to
+                         read from `./pyproject.toml`
+
+  -h, --help             Show this message and exit.
+  -V, --version          Show the version and exit.
+  -v, --verbose          Turns on debug-level logging.
 ```
 
+#### Check
+##### `--check`
+
+Using the option will not write any formatted code back to file. It will instead check whether any changes *would* be made. It returns one of three possible exit codes:
+
+**0** - indicates **no changes** would be made
+```
+$ echo 'include: "foo.txt"' | snakefmt --check -                                        
+[INFO] 1 file(s) would be left unchanged 🎉
+$ echo "Exit code: $?"
+Exit code: 0
+```
+
+**1** - indicates **changes** would be made
+```
+$ echo 'include:"foo.txt"' | snakefmt --check - 
+[INFO] 1 file(s) would be changed 😬
+$ echo "Exit code: $?"
+Exit code: 1
+```
+
+**123** - indicates there was an **internal error** such as invalid syntax
```

Comment:
I would say just error not internal error as the latter suggests there's something wrong with `snakefmt` whereas ideally there is something wrong with the Snakefile